lib/display.ts

```diff
@@ -0,0 +1,289 @@
+
+  return `${chalk.green(padding(header, padLength))} ${info}`;
+}
+
+function formatSummary(testResult: TestResult): string {
+  const depGraph: DepGraph = createFromJSON(testResult.depGraphData);
+  const pkgCount = depGraph?.getDepPkgs()?.length || 0;
+  const pathOrDepsText = `${pkgCount} dependencies`;
+  const testedInfoText = `Tested ${pathOrDepsText} for known issues`;
+  const vulnPathsText = formatVulnSummaryText(testResult.issues);
+  let summaryText = `${testedInfoText}, ${vulnPathsText}`;
+  if (testResult.issues.length === 0) {
+    summaryText = chalk.green(`✓ ${summaryText}`);
+  }
+
+  return summaryText;
+}
+
+function formatVulnSummaryText(issues: Issue[]): string {
+  if (issues.length > 0) {
+    return chalk.bold.red(`found ${issues.length} issues.`);
+  }
+
+  return "no vulnerable paths found.";
+}
+
+function getColor(severity: string): (text: string) => string {
+  let color: (text: string) => string;
+  switch (severity) {
+    case "low":
+      color = chalk.bold.blue;
+      break;
+    case "medium":
+      color = chalk.bold.yellow;
+      break;
+    case "high":
+      color = chalk.bold.red;
+      break;
+    default:
+      color = chalk.whiteBright;
+      break;
+  }
+
+  return color;
+}
+
+export function formatRemediations(res: TestResult) {
+  if (!res.docker || !res.docker.baseImageRemediation) {
+    return "";
+  }
+  const { advice, message } = res.docker.baseImageRemediation;
+  const out = [] as any[];
+
+  if (advice) {
+    for (const item of advice) {
+      out.push(formatString(item)(item.message));
+    }
+  } else if (message) {
+    out.push(message);
+  } else {
+    return "";
+  }
+  return `${out.join(BREAK_LINE)}`;
+}
+
+function formatString({ color, bold }: BaseImageRemediationAdvice) {
+  let formatter = chalk;
+  if (color && formatter[color]) {
+    formatter = formatter[color];
+  }
+  if (bold) {
+    formatter = formatter.bold;
+  }
+  return formatter;
+}
+
+function formatSuggestions(options): string {
+  if (options.isDockerUser) {
+    return "";
+  }
+
+  const dockerSuggestion: string[] = [];
+  if (options.config && options.config.disableSuggestions !== "true") {
+    const optOutSuggestions =
+      "To remove this message in the future, please run `snyk config set disableSuggestions=true`";
+    if (!options.file) {
+      dockerSuggestion.push(
+        chalk.bold.white(
+          "Pro tip: use `--file` option to get base image remediation advice.",
+        ),
+      );
+      dockerSuggestion.push(
+        chalk.bold.white(
+          `Example: $ snyk test --docker ${options.path} --file=path/to/Dockerfile`,
+        ),
+      );
+      dockerSuggestion.push(BREAK_LINE);
+      dockerSuggestion.push(optOutSuggestions);
+    } else if (!options["exclude-base-image-vulns"]) {
+      dockerSuggestion.push(
+        chalk.bold.white(
+          "Pro tip: use `--exclude-base-image-vulns` to exclude from display Docker base image vulnerabilities.",
```

Comment:
I wonder if this should be `container` instead of `Docker` (also there was another message above similarly saying `Docker image:`) CC @garethr